src/dune_rules/top_module.ml

```diff
@@ -44,6 +44,7 @@ let find_module sctx src =
          match origin with
          | Executables exes ->
            Exe_rules.rules ~sctx ~dir ~dir_contents ~scope ~expander exes
+         | Tests tests -> Test_rules.rules ~sctx ~dir_contents ~dir ~expander ~scope tests
```

Comment:
I'm introducing the `Test_rules` here, but I don't know if I should keep the old behaviour and use the `Exe_rules`. This seems more correct to me.